modules/subscriptions.php

```diff
@@ -1122,3 +1130,10 @@ function jetpack_do_subscription_form( $instance ) {
 	$output = ob_get_clean();
 	return $output;
 }
+
+jetpack_register_block( 'subscriptions', array( 'render_callback' => 'jetpack_show_subscription_block' ) );
+
```

Comment:
this needs to be registered in a file that is shared with .com

Reply:
actually, scratch that! there is not subscription-related file that is shared with .com